src/lib/IPFS/ipfs.js

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright(c) ConsenSys Software Inc.
+ * Copyright(c) https://consensys.net/
+ * MIT Licensed
+ */
+/* eslint-disable */
+import { HttpService } from '../../services/httpService.js';
+import { ERROR_LOG, errorLogger } from '../error/handler.js';
+import { isURI, toBase64 } from '../utils.js';
+import fs from 'fs';
+import FormData from 'form-data';
+
+export default class IPFS {
+  #httpClient;
+
+  constructor({ projectId, projectSecret, ipfsUrl }) {
+    this.#httpClient = new HttpService(ipfsUrl, toBase64(projectId, projectSecret));
```

Comment:
Could you use the same header injected here https://github.com/ConsenSys/infura-sdk/blob/dev/src/services/httpService.js#L21

Reply:
i'm using it, i'm creating an instance of the same class , this is the content of my header:
`headers: {
          Accept: 'application/json, text/plain, */*',
          'Content-Type': 'multipart/form-data; boundary=xxxx',
          Authorization: 'Basic xxxxxx',
          'X-Infura-User-Agent': 'infura/sdk-js 1.0.2',
          'User-Agent': 'axios/0.27.2',
        },`

Reply:
Oh yeah, my bad !